src/Discord/Parts/Channel/Channel.php

```diff
@@ -329,6 +329,80 @@ public function moveMember($member)
         return $deferred->promise();
     }
 
+    /**
+     * Mutes a member on a voice channel.
+     *
+     * @param Member|int The member to mute. (either a Member part or the member ID)
+     *
+     * @return \React\Promise\Promise
+     */
+    public function muteMember($member)
+    {
+        $deferred = new Deferred();
+
+        if (! $this->allowVoice()) {
+            $deferred->reject(new \Exception('You cannot mute a member in a text channel.'));
+
+            return $deferred->promise();
+        }
+
+        if ($member instanceof Member) {
+            $member = $member->id;
+        }
+
+        $this->http->patch(
+            "guilds/{$this->guild_id}/members/{$member}",
+            [
+                'mute' => true,
+            ]
+        )->then(
+            \React\Partial\bind_right($this->resolve, $deferred),
```

Comment:
`\React\Partial\bind([$deferred, 'resolve'])`

Reply:
will look this up in the entire code and change it wherever it needs to be changed, I based that code on Channel::deleteMessages